pipert/core/component.py

```diff
@@ -38,14 +38,20 @@ def _start(self):
             routine.start()
 
     def run(self):
+        # self.component_process = multiprocessing.Process(target=self._run)
```

Comment:
I wanted to check with itamar if that is what he meant about running each component in a separate process.
I will delete those lines because i am making it in a new branch.